libs/dh/market-participant/actors/feature-delegation/src/lib/dh-delegation-validators.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { AbstractControl, ValidationErrors } from '@angular/forms';
+import { WattRange, dayjs } from '@energinet-datahub/watt/date';
+
+export const dateCannotBeOlderThanTodayValidator =
+  () =>
+  (control: AbstractControl): ValidationErrors | null => {
+    const controlStart = control.value as string;
+    if (!controlStart) return null;
+
+    const startDate = dayjs(controlStart).toDate();
+
+    if (startDate < dayjs().startOf('day').toDate()) {
```

Comment:
suggestion: I think DayJS has a method to compare two dates. Let's use that instead.

Reply:
fixed